tool/plugin/lib/plugin.dart

```diff
@@ -135,7 +159,51 @@ Future<int> genPluginXml(BuildSpec spec, String destDir) async {
       .transform(new LineSplitter())
       .forEach((l) => dest.writeln(substitueTemplateVariables(l, spec)));
   await dest.close();
-  return dest.done;
+  return await dest.done;
+}
+
+void genTravisYml(List<BuildSpec> specs) {
+  envLine(p, i, d) =>
+      '  - IDEA_PRODUCT=$p IDEA_VERSION=$i DART_PLUGIN_VERSION=$d\n';
+  var file = new File(p.join(rootPath, '.travis.yml'));
+  String env = '';
```

Comment:
Curious why some types are declared and others (like `file` above) are not?

Reply:
I'm trying to break the habit of declaring types everywhere. It is hard.

Reply:
I'm fine w/ types in the LHS, but we should decide on one or the other for this repo. Once we've converged, there's likely a lint which we can enable to help fixup the code.